src/quo2/components/wallet/network_routing/animation.cljs

```diff
@@ -0,0 +1,116 @@
+(ns quo2.components.wallet.network-routing.animation
+  (:require [react-native.reanimated :as reanimated]))
+
+(def ^:private slider-timing 300)
+
+(defn show-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 1 slider-timing))
+
+(defn hide-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 0 slider-timing))
+
+(defn increase-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 8 slider-timing)
+  (reanimated/animate height-shared-value 40 slider-timing))
+
+(defn decrease-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 4 slider-timing)
+  (reanimated/animate height-shared-value 32 slider-timing))
```

Comment:
I still think having `slider-timing` at the end as 300 is redundant as this is the default in Reanimated library.
And when working with reanimated, we know we are working with `shared-values`.

So I think there shouldn't be confusion there.

But if you insist on keeping them that way fine.

Reply:
@OmarBasem Thanks for sharing your thoughts on it. I'm just trying to be careful when passing vars to external functions.

Another example would be when we pass an atom to another function and we perform `reset!`, `swap!` or `deref` operations on it, instead of passing the atom with name `width`, I'd pass it as `width-atom`. We are clearly using reagent but it's a way to easily differentiate and warn that we are working with an atom (for this case, with a shared value).

It's verbose and honestly I don't love it, also I questioned myself too much while naming them as this, but at the end I thought it's preferably to be clearer and that devs reading or modifying this code in the future will appreciate knowing those values are not just width or height numbers, they are shared-values.

I'd like to bring @ilmotta to this conversation, at the end Clojure is a dynamic language and my naming technique is kind of adding a "type" to the name of the vars.

@ilmotta WDYT?

Reply:
I don't hold any strong opinion on this matter, but I tend to dislike types in names, it's not very idiomatic in Clojure (and other languages too). Sometimes it's fine. For example, reanimated code often suffixes names with `*-shared-value`, which someone could also say is unnecessary.

At the end of the day, the app has tons of local state 🤷🏼, and then it's easy to get naming clashes or confuse a pure value with an atom, so I understand your choice @ulisesmac. I'm not against that but I'll probably not follow this idea either, unless the team likes it as a whole.

Reply:
@ulisesmac I understand your point of view. I don't mind either. But maybe would be better for us as a team to come to a consensus regarding this sometime to avoid inconsistencies 👍